Setting up from documentation

This is how I have been able to make it start, haven't tested it yet since I am using the PubSub emulator.

```
import { INestMicroservice, Logger } from "@nestjs/common";
import { NestFactory } from "@nestjs/core";
import { GCPubSubServer } from "@algoan/nestjs-google-pubsub-microservice";
import { AppModule } from "./app.module";
import { MicroserviceOptions } from "@nestjs/microservices";

async function bootstrap() {
	const app: INestMicroservice = await NestFactory.createMicroservice<MicroserviceOptions>(AppModule, {
		strategy: new GCPubSubServer({
			projectId: process.env.PROJECT,
			subscriptionsPrefix: `rely-${process.env.SERVICE}`
		})
	});

	await app.listen(() => Logger.log(`${process.env.SERVICE} listening`, "main"));
}
bootstrap();
```
It is quite different from the documentation which gives these errors:

```
src/main.ts:14:3 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(module: any, options?: NestApplicationOptions): Promise<INestApplication>', gave the following error.
    Argument of type '{ strategy: GCPubSubServer; }' is not assignable to parameter of type 'NestApplicationOptions'.
      Object literal may only specify known properties, and 'strategy' does not exist in type 'NestApplicationOptions'.
  Overload 2 of 2, '(module: any, httpAdapter: AbstractHttpAdapter<any, any, any>, options?: NestApplicationOptions): Promise<INestApplication>', 
gave the following error.
    Argument of type '{ strategy: GCPubSubServer; }' is not assignable to parameter of type 'AbstractHttpAdapter<any, any, any>'.
      Object literal may only specify known properties, and 'strategy' does not exist in type 'AbstractHttpAdapter<any, any, any>'.

14   strategy: new GCPubSubServer(options)
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Am I doing it wrong or is the documentation outdated?

--- a/packages/google-pubsub-microservice/README.md
+++ b/packages/google-pubsub-microservice/README.md
@@ -36,7 +36,7 @@ async function bootstrap() {
     subscriptionsPrefix: 'test-app',
   }
 
-  const app: INestMicroservice = await NestFactory.create(AppModule, {
+  const app: INestMicroservice = await NestFactory.createMicroservice(AppModule, {
     strategy: new GCPubSubServer(options)
   })
 

--- a/packages/google-pubsub-microservice/package.json
+++ b/packages/google-pubsub-microservice/package.json
@@ -39,13 +39,13 @@
     "url": "https://github.com/algoan/nestjs-components/issues"
   },
   "peerDependencies": {
-    "@algoan/pubsub": "^4.3.1",
-    "@nestjs/common": "^7.6.15",
-    "@nestjs/core": "^7.6.15",
-    "@nestjs/microservices": "^7.6.15"
+    "@algoan/pubsub": ">=4.x.x",
+    "@nestjs/common": ">=7.x.x",
+    "@nestjs/core": ">=7.x.x",
+    "@nestjs/microservices": ">=7.x.x"
   },
   "devDependencies": {
-    "@algoan/pubsub": "4.4.1",
+    "@algoan/pubsub": "4.6.0",
     "@nestjs/common": "7.6.15",
     "@nestjs/core": "7.6.15",
     "@nestjs/microservices": "7.6.15",
@@ -55,6 +55,6 @@
     "jest": "^26.6.3",
     "reflect-metadata": "^0.1.13",
     "rxjs": "^6.6.7",
-    "typescript": "^4.2.3"
+    "typescript": "^4.2.4"
   }
 }

